API/Endpoints/DashboardEndpoints.cs

```diff
@@ -0,0 +1,70 @@
+using System.Security.Claims;
+using API.Configuration;
+using API.Endpoints.Shared;
+using Asp.Versioning;
+using Asp.Versioning.Builder;
+using Core.Dashboards.Contracts;
+using Core.Dashboards.Models;
+using Core.Shared;
+using Microsoft.AspNetCore.Http.HttpResults;
+
+namespace API.Endpoints;
+
+public static class DashboardEndpoints
+{
+    public static WebApplication UseDashboardEndpoints(this WebApplication app)
+    {
+        ApiVersionSet apiVersionSet = app.NewApiVersionSet()
+            .HasApiVersion(new ApiVersion(1))
+            .ReportApiVersions()
+            .Build();
+
+        var dashboardV1Group = app.MapGroup("v{version:apiVersion}/dashboard").WithApiVersionSet(apiVersionSet)
+            .WithTags("Dashboard");
+
+        //Get exercises in timed_session
+        dashboardV1Group.MapGet("/timedSession/{sessionId:int}", async Task<Results<Ok<GetExercisesInSessionCombinedInfo>, NotFound, BadRequest>> (int sessionId, ClaimsPrincipal principal,
```

Comment:
Make generic GET endpoint on "/{id}" and distinguish on service level if the session is classroom or timed session.

Combines this in "/classSession" endpoint